applications/GeoMechanicsApplication/tests/test_helper.py

```diff
@@ -419,6 +420,97 @@ def find_closest_index_greater_than_value(input_list, value):
     return None
 
 
+def are_values_almost_equal(expected: Any, actual: Any, abs_tolerance: float = 1e-7) -> bool:
+    """
+    Checks whether two values are almost equal.
+
+    Args:
+        - expected (Any): Expected value.
+        - actual (Any): Actual value.
+
+    Returns:
+        - True if the values are almost equal, False otherwise.
+
+    """
+    # check if the value is a dictionary and check the dictionary
+    if isinstance(expected, dict):
+        # check if the value is a dictionary and check the dictionary
+        if not are_dictionaries_almost_equal(expected, actual):
+            return False
+    elif isinstance(expected, str):
+        # check if the value is a string and compare the strings
+        if expected != actual:
+            return False
```

Comment:
Also here, we can get rid of a branch and a redundant comment:
```suggestion
        return expected == actual
```